File: CONCEPTS/anonymous1.py
import pandas as pd
# Create a sample data frame
df = pd.DataFrame({
    'id': [1, 2, 3],
    'value1': [10, 20, 30],
    'value2': [2, 5, 3]
})
# Use assign() with a lambda function to create a new column
df_modified = df.assign(
    new_value=lambda x: x['value1'] * 2 + x['value2']
)
# Print the modified data frame
print(df_modified)

File: CONCEPTS/anonymous1.R
library(dplyr) 
library(purrr)
# Create a sample data frame
df <- data.frame(
  id = 1:3, 
  value1 = c(10, 20, 30),
  value2 = c(2, 5, 3)
)

# Use an anonymous function with mutate to create a new column
# The anonymous function calculates (value1 * 2) + value2   

df_modified_purrr <- df %>%
  mutate(new_value = pmap_dbl(list(value1, value2), ~ .x * 2 + .y))

print(df_modified_purrr)

